docs/quickstart.md

```diff
@@ -0,0 +1,108 @@
+## Quickstart
+
+This tutorial assumes you are starting fresh and have no existing collectd agent running on your instance.
+
+#### Step 1: Download and install the agent
+
+```sh
+curl -sSL https://dl.signalfx.com/signalfx-agent.sh > /tmp/signalfx-agent.sh
+sudo sh /tmp/signalfx-agent.sh YOUR_SIGNALFX_API_TOKEN
+```
+
+Your SignalFx API Token can be obtained from the Organization->Access Token tab in [SignalFx](https://app.signalfx.com)
+
+More detailed installation steps to install via a config management tool or using a containerized agent can be found [here](../README.md#installation)
+
+#### Step 2: Configuration
+
+The default configuration file should be located at /etc/signalfx/agent.yaml
+
+Here is an example of the configuration file - 
+
+```
+---
+# *Required* The access token for the org that you wish to send metrics to.
+signalFxAccessToken: {"#from": "/etc/signalfx/token"}
```

Comment:
If you're going to tell people to use this config, it should be explicitly stated that they should put the access token in the specified file since they will probably not be immediately familiar with the remote config syntax.

Reply:
thanks, good catch. I'll add that to the docs.